src/test_ops.rs

```diff
@@ -633,6 +633,9 @@ l => FAIL
 l ( 1 2 ) 1 => FAIL
 l ( 1 2 3 ) => 1
 l 1 => 0
+l null => null
```

Comment:
I would recommend against using`null`, as it's not used anywhere else around CLVM (just use `0`).

Reply:
it is used on the allocator, which has a member function `null()`, but I see your point. The reason I ended up using it here was because the first version of this test framework didn't support decimal numbers.

Reply:
making this change is rather big, and I'm not 100% sold on it. right now there is a certain simplicity of having a special token for what `Allocator::null()` returns, rather than depending on (the quite involved) logic of parsing integers. On the other hand, that integer parsing better be correct as well..

Reply:
Ah, I see what motivates this now. My only concern is that it might be confusing to someone looking at it coming from a clvm use background rather than a clvm implementation background. A compromise might be to look for the string `0` (instead of `null`) and call `allocator.null`, and also assert that it matches `0` as a string.

Reply:
yeah, I like that